toppra/solverwrapper/cy_seidel_solverwrapper.pyx

```diff
@@ -432,13 +436,17 @@ cdef class seidelWrapper:
             The geometric path.
         path_discretization: array
             The discretized path positions.
+        solve_lp1d: int
+            if solve_lp1d > 0, Solve 1D LP instead of 2D LP when possible (x_min == x_max)
```

Comment:
```suggestion
            if solve_lp1d > 0, solve 1D LP instead of the full 2D LP when possible: when x_min == x_max, solve for u as the variable.
```